default.nix

```diff
@@ -76,7 +76,7 @@ rec {
     random
     template
     tls
-    vault
+    p.vault
```

Comment:
@Benjile : can you explain a little your change?

Reply:
if you build terraform, nix will build the vault hashicorp standard package and not the terraform plugin because this name (vault) is defined a few lines above
`51   inherit (pkgs) kubectl stern vault docker-compose cfssl kompose$
 52                  yq jq gopass kubectx awscli direnv cue go gnupg curl kustomize;$
`

Reply:
```
nix-build -A terraform
/nix/store/cwyh65anxk2992p9vr9gxpn4gd7c119g-terraform-0.12.26

cat /nix/store/cwyh65anxk2992p9vr9gxpn4gd7c119g-terraform-0.12.26/bin/terraform
#! /nix/store/xfbmj7sl2ikicym9x3yq7cms5qx1w39k-bash-4.4-p23/bin/bash -e
export NIX_TERRAFORM_PLUGIN_DIR='/nix/store/j9m5vpg4pb1pfzix5qsimvzm08aj3frq-tf-plugin-env/bin'
exec "/nix/store/jpvqd1crm09bci10b5r338nwbv22fn16-terraform-0.12.26/bin/terraform"  "$@"

ll /nix/store/j9m5vpg4pb1pfzix5qsimvzm08aj3frq-tf-plugin-env/bin
total 64K
lrwxrwxrwx 1 benjamin benjamin 108 Jan  1  1970 terraform-provider-aws_v2.59.0 -> /nix/store/kl32r7wf6asyw293mb6m3svi517fcprb-terraform-provider-aws-2.59.0/bin/terraform-provider-aws_v2.59.0
lrwxrwxrwx 1 benjamin benjamin 118 Jan  1  1970 terraform-provider-concourse_v5.8.0 -> /nix/store/7m8zdslacih1v8x1bwlqap7c3yngqwk0-terraform-provider-concourse-5.8.0/bin/terraform-provider-concourse_v5.8.0
lrwxrwxrwx 1 benjamin benjamin 116 Jan  1  1970 terraform-provider-external_v1.2.0 -> /nix/store/lg8jb5zljrdsjhnic5g6fcwar8av9ama-terraform-provider-external-1.2.0/bin/terraform-provider-external_v1.2.0
lrwxrwxrwx 1 benjamin benjamin 130 Jan  1  1970 terraform-provider-flexibleengine_v1.12.1 -> /nix/store/ii9vw7fz4bgxrkh7cgc1gf9fn71dn4vm-terraform-provider-flexibleengine-1.12.1/bin/terraform-provider-flexibleengine_v1.12.1
lrwxrwxrwx 1 benjamin benjamin 108 Jan  1  1970 terraform-provider-http_v1.2.0 -> /nix/store/9gg02zlz1qda8xk5fmqv9sq5fdzhxprs-terraform-provider-http-1.2.0/bin/terraform-provider-http_v1.2.0
lrwxrwxrwx 1 benjamin benjamin 124 Jan  1  1970 terraform-provider-huaweicloud_v1.14.0 -> /nix/store/3n4w80amv4jnns4qlwi6cac4c980r08m-terraform-provider-huaweicloud-1.14.0/bin/terraform-provider-huaweicloud_v1.14.0
lrwxrwxrwx 1 benjamin benjamin 112 Jan  1  1970 terraform-provider-k8sraw_v0.2.0 -> /nix/store/8xp6i0dk4kh1v7qq6z7iy6l9mxp9p6a3-terraform-provider-k8sraw-0.2.0/bin/terraform-provider-k8sraw_v0.2.0
lrwxrwxrwx 1 benjamin benjamin 118 Jan  1  1970 terraform-provider-keycloak_v1.17.1 -> /nix/store/jf5jm1w3l89pdln99s8grhd5savbqwaq-terraform-provider-keycloak-1.17.1/bin/terraform-provider-keycloak_v1.17.1
lrwxrwxrwx 1 benjamin benjamin 122 Jan  1  1970 terraform-provider-kubernetes_v1.11.1 -> /nix/store/si5y1dshpwrs7ycwjp9nlp2pi79dh0wa-terraform-provider-kubernetes-1.11.1/bin/terraform-provider-kubernetes_v1.11.1
lrwxrwxrwx 1 benjamin benjamin 110 Jan  1  1970 terraform-provider-local_v1.4.0 -> /nix/store/wjsa2by9b0lg3yay5n35knil5zrd89mc-terraform-provider-local-1.4.0/bin/terraform-provider-local_v1.4.0
lrwxrwxrwx 1 benjamin benjamin 120 Jan  1  1970 terraform-provider-openstack_v1.27.0 -> /nix/store/z578v312nd26jk934ab3797nqbs4vprq-terraform-provider-openstack-1.27.0/bin/terraform-provider-openstack_v1.27.0
lrwxrwxrwx 1 benjamin benjamin 116 Jan  1  1970 terraform-provider-rancher2_v1.8.3 -> /nix/store/h7l3vd0f0k1i1i8lwhad6nz0xbx54mix-terraform-provider-rancher2-1.8.3/bin/terraform-provider-rancher2_v1.8.3
lrwxrwxrwx 1 benjamin benjamin 112 Jan  1  1970 terraform-provider-random_v2.2.1 -> /nix/store/nz31f6qpvraz74461bkqd18hp88ylqxa-terraform-provider-random-2.2.1/bin/terraform-provider-random_v2.2.1
lrwxrwxrwx 1 benjamin benjamin 116 Jan  1  1970 terraform-provider-template_v2.1.2 -> /nix/store/6kcsjbj62afgw4i6dajrz2czcbn3xzl2-terraform-provider-template-2.1.2/bin/terraform-provider-template_v2.1.2
lrwxrwxrwx 1 benjamin benjamin 106 Jan  1  1970 terraform-provider-tls_v2.1.1 -> /nix/store/ckzybsnv39hwrs3hz43ql78pl98lky6j-terraform-provider-tls-2.1.1/bin/terraform-provider-tls_v2.1.1
lrwxrwxrwx 1 benjamin benjamin  65 Jan  1  1970 vault -> /nix/store/jrjklk0hwp1hmg2nabbvcy6ggiy7bdn0-vault-1.4.2/bin/vault

nix/store/jrjklk0hwp1hmg2nabbvcy6ggiy7bdn0-vault-1.4.2/bin/vault
Usage: vault <command> [args]

Common commands:
    read        Read data and retrieves secrets
    write       Write data, configuration, and secrets
    delete      Delete secrets and configuration
    list        List data or secrets
    login       Authenticate locally
    agent       Start a Vault agent
    server      Start a Vault server
    status      Print seal and HA status
    unwrap      Unwrap a wrapped secret

Other commands:
    audit          Interact with audit devices
    auth           Interact with auth methods
    debug          Runs the debug command
    kv             Interact with Vault's Key-Value storage
    lease          Interact with leases
    namespace      Interact with namespaces
    operator       Perform operator-specific tasks
    path-help      Retrieve API help for paths
    plugin         Interact with Vault plugins and catalog
    policy         Interact with policies
    print          Prints runtime configurations
    secrets        Interact with secrets engines
    ssh            Initiate an SSH session
    token          Interact with tokens
```